index.html

```diff
@@ -0,0 +1,213 @@
+            </div>
+            <div class="content row justify-content-center pt-5">
+                <div class="col-md d-flex flex-sm-column flex-row align-items-center justify-content-center card">
+                    <div class="icons">
+                        <i class="fa fa-cart-shopping"></i>
+                    </div>
+                    <div>
+                        <h4 class="first-color card-title">Card</h4>
+                    </div>
+                    <div>
+                        <p class="third-color card-p">The Wakhjir brand makes handcrafted women's and children's clothing, handbags, and shoes suitable for evening events like weddings, engagement parties, and weddings. The brand creates the most genuine and high-quality accessories by blending sophisticated vintage styles with modern patterns on natural fabrics like silk, cotton, georgette, woollen, linen, and leather. With Wakhjir outfits and accessories perfect every aspect of your look.</p>
+                    </div>
+                </div>
+                <div class="col-md d-flex flex-sm-column flex-row align-items-center justify-content-center card">
+                    <div class="icons">
+                        <li class="fa fa-bag-shopping"></li>
+                    </div>
+                    <div>
+                        <h4 class="first-color card-title">Bag</h4>
+                    </div>
+                    <div>
+                        <p class="third-color card-p">The Wakhjir brand makes handcrafted women's and children's clothing, handbags, and shoes suitable for evening events like weddings, engagement parties, and weddings. The brand creates the most genuine and high-quality accessories by blending sophisticated vintage styles with modern patterns on natural fabrics like silk, cotton, georgette, woollen, linen, and leather. With Wakhjir outfits and accessories perfect every aspect of your look.</p>
+                    </div>
+                </div>
+                <div class="col-md d-flex flex-sm-column flex-row align-items-center justify-content-center card">
+                    <div class="icons">
+                        <li class="fa-solid fa-bag-shopping"></li>
+                    </div>
+                    <div>
+                        <h4 class="first-color card-title">Bag</h4>
+                    </div>
+                    <div>
+                        <p class="third-color card-p">The Wakhjir brand makes handcrafted women's and children's clothing, handbags, and shoes suitable for evening events like weddings, engagement parties, and weddings. The brand creates the most genuine and high-quality accessories by blending sophisticated vintage styles with modern patterns on natural fabrics like silk, cotton, georgette, woollen, linen, and leather. With Wakhjir outfits and accessories perfect every aspect of your look.</p>
+                    </div>
```

Comment:
- [ OPTIONAL] I would recommend to minimize the length of your describtion, because in mobile version it is making it look vague. consider the size used in the provided  behance design and the fonts to make yours more beautiful.
![image](https://user-images.githubusercontent.com/43877689/234565119-d81986db-2798-422e-8ab9-865fa636b86d.png)